hack/verify-gofmt.sh

```diff
@@ -56,7 +53,7 @@ find_files() {
 # formatting (e.g., a file does not parse correctly). Without "|| true" this
 # would have led to no useful error message from gofmt, because the script would
 # have failed before getting to the "echo" in the block below.
-diff=$(find_files | xargs "${gofmt}" -d -s 2>&1) || true
+diff=$(find_files | xargs gofmt -d -s 2>&1) || true
```

Comment:
Have we guaranteed the correct version of gofmt is in PATH by this point?

Reply:
`kube::golang::setup_env` calls `kube::golang::verify_go_version` internally (and documents this).

we just don't need to use `which` (path lookup) OR query bazel now, we can just use path.